notebooks/community/ml_ops/stage2/get_started_vertex_training_xgboost.ipynb

```diff
@@ -29,7 +29,7 @@
         "id": "title:generic,gcp"
```

Comment:
<p>Add open in Colab link</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/411/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>